_posts/2023-05-02-2-opslag-in-de-database-en.markdown

```diff
@@ -27,9 +27,12 @@ Lastly, this means of communication is used for communication for which the inte
 *Legal Ground b, minimal execution membership agreement*  
 In the case that the primary correspondence mail address is unavailable, Inter-*Actief* uses the mailing address of the members. The mailing address is therefore saved by Inter-*Actief*.
 
+*Legal Ground a, receive association magazine*  
+The postal address is also used to send the association magazine to, if a person has given explicit permission for this.
```

Comment:
Maybe generalize that we send post to the address, since next to the magazine we might also send constitution cards or Christmas cards?